src/Fixer/Phpdoc/PhpdocSeparationFixer.php

```diff
@@ -92,6 +124,27 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
         }
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    protected function createConfigurationDefinition(): FixerConfigurationResolverInterface
+    {
+        return new FixerConfigurationResolver([
+            (new FixerOptionBuilder('groups', 'Sets of annotation types to be grouped together.'))
+                ->setAllowedTypes(['string[][]'])
+                ->setDefault(TagComparator::DEFAULT_GROUPS)
+                ->getOption(),
```

Comment:
What is the behavior of the rule in case a tag is part of several groups? Should we validate the option to prevent that?

Reply:
The behavior depends on the order of the tags that are processed by the fixer. And yes, I think we should validate the option to prevent the tag from belonging to several groups.

Reply:
Validation added.